### PR TITLE
TR-2017: Unable to save and publish changes to template options

### PR DESCRIPTION
**Pull request by bkemper**
### What Changed
 - The PUT request to save a template is made with all the template params instead of just content.

### How To Test
 - Open a draft template in the editor
 - Open the "template settings" tab
 - Modify any of the fields (easiest with click & open options)
 - Click "Save Settings" button
 - Click "Save as Draft" button
 - Refresh the page to confirm changes persisted

### CODE REVIEW

**Comment by nicklemmon** [comment]
@bkemper  Hmm, so if I don't click 'Update Settings' and just click 'Save Draft' I lose my changes. Is that expected?

File: src/pages/templates/components/settings/SettingsForm.js [hunk 1]
```diff
@@ -12,39 +12,49 @@ import styles from './SettingsForm.module.scss';
...
+    return updateDraft(
+      {
+        id: draft.id,
+        name: values.name,
+        description: values.description,
+        // must include other content parts to avoid work in progress
+        content: { ...values.content, amp_html, html, text },
+        options: values.options,
+        // this value is only editable and present when template is not assigned to a subaccount
+        shared_with_subaccounts: draft.subaccount_id ? false : values.shared_with_subaccounts,
```
[the remark this answers is not in the document]
Comment (bkemper) [reply src/pages/templates/components/settings/SettingsForm.js]: Agreed.

**Comment by bkemper** [comment]
@nicklemmon Ready for final review.

**Comment by nicklemmon** [comment]
@bkemper Very confused - the tests pass locally, and only one of the two Travis CI runs are unsuccessful. A caching issue?

Ideally we wouldn't have both of these runs happening on every PR anyways.

**Pull request merged**